app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::Base
+  helper_method :current_user
+
+  def current_user
+    user_id = params[:user_id]
+    User.find_by(id: user_id)
+  end
 end
```

Comment:
- [ ] Great work adding a current_user method. It will be great to ensure that your current_user method returns the first user in your database. The way your code is working at the moment, the current_user will be the user who has the same id as the one in the parameters. As you can see below, ⬇️  the current user should be the first user in your database.
![Screenshot 2023-10-14 at 14 22 23](https://github.com/SanaTameem/blog-app/assets/32361963/410600ad-b7c9-45b6-9990-2c11f759b6c1)

### EXAMPLE
```
def current_user
  User.first
end
```